packages/sdk-components-react-radix/src/collapsible.tsx

```diff
@@ -35,3 +36,41 @@ export const CollapsibleTrigger = forwardRef<
 export const CollapsibleContent: ForwardRefExoticComponent<
   Omit<ComponentPropsWithRef<typeof Content>, "asChild">
 > = Content;
+
+/* BUILDER HOOKS */
+
+const namespace = "@webstudio-is/sdk-components-react-radix";
+
+// For each CollapsibleContent component within the selection,
+// we identify its closest parent Collapsible component
+// and update its open prop bound to variable.
+export const hooksCollapsible: Hook = {
+  onNavigatorUnselect: (context, event) => {
+    for (const instance of event.instanceSelection) {
+      if (instance.component === `${namespace}:CollapsibleContent`) {
+        const collapsible = getClosestInstance(
+          event.instanceSelection,
+          instance,
+          `${namespace}:Collapsible`
+        );
+        if (collapsible) {
+          context.setPropVariable(collapsible.id, "open", false);
+        }
+      }
+    }
+  },
+  onNavigatorSelect: (context, event) => {
+    for (const instance of event.instanceSelection) {
```

Comment:
`instanceSelection` vs `selectedInstances`

If its a list of instances that are selected the later seems semantically closer to what it is

Reply:
instanceSelector -> instanceSelection
No the list of multi select

Reply:
yes, selectedInstances is a list of selected instances semantically

Reply:
selected instances semantically is multi select
here it is a path from current to root

Reply:
ah so instanceSelection is a path of ordered instances from current to the root? Got it now, wouldn't understand it from naming at all. Need a name that reflects exactly this